ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallDataExtensions.kt

```diff
@@ -0,0 +1,51 @@
+package com.revenuecat.purchases.ui.revenuecatui
+
+import android.util.Log
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.core.os.ConfigurationCompat
+import androidx.core.os.LocaleListCompat
+import com.revenuecat.purchases.paywalls.PaywallData
+import java.util.Locale
+
+@Composable
+@ReadOnlyComposable
+internal fun PaywallData.getColors(): PaywallData.Configuration.Colors {
+    return config.colors.dark?.takeIf { isSystemInDarkTheme() } ?: config.colors.light
+}
+
+@Composable
+@ReadOnlyComposable
+internal fun PaywallData.localizedConfig(): PaywallData.LocalizedConfiguration? {
+    val locale = getLocale()
+    val configForLocale = configForLocale(locale)
+    if (configForLocale == null) {
+        Log.e("RevenueCatUI", "No configuration found for locale $locale")
```

Comment:
We'll probably want to extract this to avoid hardcoding `RevenueCatUI` everywhere, right?

Reply:
Right, did that in a followup PR #1233